ebpfapi/ebpfapi.vcxproj

```diff
@@ -114,6 +133,21 @@
       <AdditionalDependencies>mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <AdditionalLibraryDirectories>$(OutDir)</AdditionalLibraryDirectories>
     </Link>
+    <ResourceCompile>
+      <AdditionalIncludeDirectories>$(OutDir)</AdditionalIncludeDirectories>
+    </ResourceCompile>
+    <CustomBuildStep>
+      <Command>
+      </Command>
+      <Outputs>
+      </Outputs>
+      <Message />
+    </CustomBuildStep>
+    <PreBuildEvent>
+      <Command>cd $(OutDir)
+powershell $(SolutionDir)scripts\generate-commitid.ps1
+cd ..</Command>
```

Comment:
can you pass $(OutDir) as an argument to the script rather than requiring it to be run from within that directory?